exercises/practice/crypto-square/crypto_square_test.cpp

```diff
@@ -5,82 +5,51 @@
+    REQUIRE("" == crypto_square::cipher("").normalized_cipher_text());
 }
 
 #if defined(EXERCISM_RUN_ALL_TESTS)
-TEST_CASE("normalize_spaces")
-{
-    REQUIRE("hithere" == crypto_square::cipher("Hi there").normalize_plain_text());
-}
-
-TEST_CASE("normalize_numbers")
-{
-    REQUIRE("123go" == crypto_square::cipher("1, 2, 3 GO!").normalize_plain_text());
-}
-
-TEST_CASE("plain_text_empty")
-{
-    const std::vector<std::string> expected{};
-
-    const auto actual = crypto_square::cipher("").plain_text_segments();
-
-    REQUIRE(expected == actual);
-}
-
-TEST_CASE("plain_text_4_characters")
-{
-    const std::vector<std::string> expected{"ab", "cd"};
-
-    const auto actual = crypto_square::cipher("Ab Cd").plain_text_segments();
-
-    REQUIRE(expected == actual);
+TEST_CASE("normalization results in empty plaintext",
+          "[aad04a25-b8bb-4304-888b-581bea8e0040]") {
+    REQUIRE("" ==
+            crypto_square::cipher("... --- ...").normalized_cipher_text());
 }
 
-TEST_CASE("plain_text_9_characters")
-{
-    const std::vector<std::string> expected{"thi", "sis", "fun"};
-
-    const auto actual = crypto_square::cipher("This is fun!").plain_text_segments();
-
-    REQUIRE(expected == actual);
+TEST_CASE("Lowercase", "[64131d65-6fd9-4f58-bdd8-4a2370fb481d]") {
+    REQUIRE("a" == crypto_square::cipher("A").normalized_cipher_text());
 }
 
-TEST_CASE("plain_text_segments_from_phrase")
-{
-    const std::vector<std::string> expected{"ifmanwas", "meanttos", "tayonthe", "groundgo", "dwouldha", "vegivenu", "sroots"};
-
-    const auto actual = crypto_square::cipher("If man was meant to stay on the ground, god would have given us roots.").plain_text_segments();
-
-    REQUIRE(expected == actual);
-}
-
-TEST_CASE("cipher_text_empty_phrase")
-{
-    REQUIRE("" == crypto_square::cipher("").cipher_text());
+TEST_CASE("Remove spaces", "[63a4b0ed-1e3c-41ea-a999-f6f26ba447d6]") {
+    REQUIRE("b" == crypto_square::cipher("  b ").normalized_cipher_text());
 }
 
-TEST_CASE("cipher_text_long_phrase")
-{
-    REQUIRE("imtgdvsfearwermayoogoanouuiontnnlvtwttddesaohghnsseoau" ==
-        crypto_square::cipher("If man was meant to stay on the ground, god would have given us roots.").cipher_text());
+TEST_CASE("Remove punctuation", "[1b5348a1-7893-44c1-8197-42d48d18756c]") {
+    REQUIRE("1" == crypto_square::cipher("@1,%!").normalized_cipher_text());
 }
 
-TEST_CASE("normalized_cipher_text_empty")
-{
-    REQUIRE("" == crypto_square::cipher("").normalized_cipher_text());
+TEST_CASE("9 character plaintext results in 3 chunks of 3 characters",
+          "[8574a1d3-4a08-4cec-a7c7-de93a164f41a]") {
+    REQUIRE("tsf hiu isn" ==
+            crypto_square::cipher("This is fun!").normalized_cipher_text());
 }
 
-TEST_CASE("normalized_cipher_text_fun")
-{
-    REQUIRE("tsf hiu isn" == crypto_square::cipher("This is fun!").normalized_cipher_text());
+TEST_CASE(
+    "8 character plaintext results in 3 chunks, the last one with a trailing "
```

Comment:
Currently, line breaks in the "name" of the exercise will break the test runner, can you keep them on one line until I fix this behavior?